docs/YUKA_GUIDE.md

```
@@ -0,0 +1,440 @@
+# Yuka.js - Complete Guide for Otter Empire
+
+**Source**: https://github.com/Mugen87/yuka
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)